GVFS/GVFS.Service/IRepoRegistry.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+
+namespace GVFS.Service
+{
+    public interface IRepoRegistry
+    {
+        bool TryRegisterRepo(string repoRoot, string ownerSID, out string errorMessage);
+        bool TryDeactivateRepo(string repoRoot, out string errorMessage);
+        bool TryGetActiveRepos(out List<RepoRegistration> repoList, out string errorMessage);
+        bool TryRemoveRepo(string repoRoot, out string errorMessage);
+        void AutoMountRepos(int sessionId);
+        void TraceStatus();

```

Comment:
I think this method should not be part of this interface and should be a private method in the `RepoRegistry` class and called in the `AutoMountRepos` method.  It doesn't seem like something that should be exposed to user of the class.

Reply:
Right now, `TraceStatus()` is a `public` method in `RepoRegistry`. Moving it to `private` would not be relevant to this PR. May be file a follow-up issue?

Reply:
I think it is relevant to the PR because it is creating the interface that will be used and I don't think the `TraceStatus` method belongs in this interface.